### PR TITLE
Bring soak test Kubernetes YAML up to date

### PR DESCRIPTION
**Pull request by sophieyfang**

### CODE REVIEW

**Comment by sophieyfang** [comment]
@qingling128  i am going to test the new config in our tests, i will update the status in our internal bug ticjet

File: agents.yaml [hunk 1]
```diff
@@ -388,59 +397,54 @@ data:
...
+    # This configuration file for Fluentd is used
+    # to watch changes to Kubernetes pod log files live in the
+    # directory /var/log/pods/NAMESPACE_NAME_UID. The file name
+    # is used to capture the pod namespace, name and uid. These
+    # logs are then submitted to Google Cloud Logging with a
+    # local_resource_id 'k8s_pod.<NAMESPACE_NAME>.<POD_NAME>'
+    # which assumes the installation of the cloud-logging plug-in.
+    <source>
+      @type tail
+      {{ if .SystemOnlyLogging }}
```
Comment (StevenYCChou) [inline agents.yaml]: Can template be used here?

File: agents.yaml [hunk 2]
```diff
@@ -388,59 +397,54 @@ data:
...
+      path /var/log/pods/kube-system_*/*,/var/log/pods/istio-system_*/*,/var/log/pods/knative-serving_*/*,/var/log/pods/gke-system_*/*,/var/log/pods/config-management-system_*/*,
+      {{ else }}
+       path /var/log/pods/*/*.log
```
Comment (StevenYCChou) [inline agents.yaml]: Use path in `else` condition would enable us to get all the logs, not only system logs.

nit: space is not aligned, so I added a suggestion here.
 
```suggestion
      path /var/log/pods/*/*.log
```

File: agents.yaml (hunk 1 above)
Comment (qingling128) [reply agents.yaml]: +1. We should keep the `else` part only.

File: agents.yaml [hunk 3]
```diff
@@ -628,17 +605,7 @@ data:
...
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: logging-agent-output-config
-  namespace: stackdriver-agents
-data:
-  google-fluentd.conf: |-
-    @include config.d/*.conf
-
+  output.conf: |-
```
Comment (qingling128) [inline agents.yaml]: As discussed offline, let's give these `.conf` file prefixed order. This ensures the order. e.g.:

```
1.containers.input.conf
2.pods.input.conf
3.system.input.conf
4.knative.input.conf
5.monitoring.conf
6.output.conf
```

File: agents.yaml (hunk 1 above)
Comment (sophieyfang) [reply agents.yaml]: Nice catch

**Pull request merged**